### PR TITLE
openshift_logging reconcile downstream with upstream

### PR DESCRIPTION
**Pull request by jcantrill**
blocked by https://github.com/openshift/origin-aggregated-logging/pull/366

### CODE REVIEW

File: roles/openshift_logging/templates/es.j2 [hunk 1]
```diff
@@ -95,6 +95,13 @@ spec:
               readOnly: true
             - name: elasticsearch-storage
               mountPath: /elasticsearch/persistent
+          readinessProbe:
```
Comment (ewolinetz) [inline roles/openshift_logging/templates/es.j2]: this is captured here https://github.com/openshift/openshift-ansible/pull/3733 but was lost... i'll merge the original one if you want to remove it here

Comment (ewolinetz) [inline roles/openshift_logging/templates/es.j2]: it also looks like that is PR is blocked waiting on another PR

**Comment by ewolinetz** [comment]
Can you rebase please? #3733 was merged last night

**Comment by jcantrill** [comment]
Closing this PR as I altered the location where Docker is placing files.

**Pull request closed without merging**